prototypes/factory.lua

```diff
@@ -221,7 +221,7 @@ for i=10,99 do
 	data:extend(factory_1("-s" .. i, "-s", "-s" .. i, false, 1, F.."/graphics/factory/factory-1-combined.png"))
 end
 data:extend(factory_1("-i", "-i", "", false, 1, F.."/graphics/factory/factory-1-combined.png"))
-	
```

Comment:
no whitespaces changes I guess. They aren't great for reducing the diff size, better do that in a seperate PR to be able to track down those changes in history.

Reply:
My editor did that automatically to all the files, I went back and undid this before pushing but I guess I missed some spots.

Reply:
ok :)

Thanks for the report! Appreaciated :+1: